cves/kernel/CVE-2021-20317.yml

```diff
@@ -55,7 +54,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  There was a corrupted node in the timer tree that caused a tasked to be missed 
```

Comment:
Grammar (minor nitpicks):
I believe this line would read better if it ended with "caused a task to be missed and prevented from being run"
Also on line 59, "This created an issue when gathering the required functionality..." is a bit more clear in my opinion

Reply:
Thanks for letting me know, issue fixed.